composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "mondu_gmbh/magento2-payment",
   "description": "Mondu payment method for magento 2",
   "type": "magento2-module",
-  "version": "2.3.4-RC2",
+  "version": "2.3.5",
```

Comment:
Lets change the version to 2.3.3 ( since last released version that's available in composer is 2.2.2 )

Reply:
@tikohov20 updated